plugins/creyaml.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: creyaml
+spec:
+  version: v0.0.1
+  homepage: https://github.com/sahil-lakhwani/kubectl-creyaml
+  shortDescription: Generate Custom Resource Example Yaml
```

Comment:
this doesn't need to be all-capitalized.

> Generate custom resource YAML manifest

is ok

Reply:
changed.